pom.xml

```diff
@@ -46,23 +40,6 @@
         </repository>
     </repositories>
 
-    <pluginRepositories>
-        <!-- The order of definitions matters. Explicitly defining central here to make sure it has the highest priority. -->
-        <pluginRepository>
-            <id>central</id>
-            <url>https://repo.maven.apache.org/maven2</url>
-            <snapshots>
```

Comment:
It is not suggested to remove the central repo, as that will cause the project build fail on searching central repo using http instead of https.

Reply:
Fixed